package.json

```diff
@@ -13,6 +13,8 @@
   "dependencies": {
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
+    "express-validator": "^7.0.1",
+    "http-status-code": "^2.1.0",
```

Comment:
What are you using `http-status-code` for?

Reply:
Funny story. I was going to use it because in that course, the instructor said it was better than using the numbers themselves but then it was giving me so unnecessary problems and I tried the numbers and they weren't doing that so I decided to not use it. I've removed the package and this can be seen on 6af1fdf